app/forms/applicant_form.rb

```diff
@@ -3,32 +3,18 @@ class ApplicantForm
   include ActiveModel::Attributes
   include SessionPersistable
 
-  PERMANENT_ATTRIBUTES = %i[passporting over_60 employed].freeze
-  CONTINGENT_ATTRIBUTES = %i[partner_over_60 partner_employed].freeze
+  PROCEEDING_TYPES = { domestic_abuse: "DA001", other: "SE003" }.freeze
 
-  ATTRIBUTES = PERMANENT_ATTRIBUTES + CONTINGENT_ATTRIBUTES.freeze
+  PROCEEDING_ATTRIBUTE = %i[proceeding_type].freeze
+  PERMANENT_ATTRIBUTES = %i[passporting over_60 employed partner].freeze
```

Comment:
I don't think `PERMANENT_ATTRIBUTES` is the right name here - all attributes listed are permanent. What sets these ones apart now is that they are _boolean_.